spec/values/colors/equality.hrx

```diff
@@ -0,0 +1,251 @@
+<===> true/legacy/same_space/rgb/no_none/input.scss
+a {b: #abcdef == #abcdef}
```

Comment:
When Sass parses `#abc`, does it internally store it as if it parsed `#aabbcc`?

Currently `#abc == #aabbccff` returns `true`, per the spec changes this will now return `false`, or no?

And if it now returns `false`, is this covered by the `false/legacy/same_space/rgb/one_none` test?

Reply:
All colors are stored as four floating-point numbers (the three color space channels and the alpha channel) as well as the color space name. `#abc`, `#aabbcc`, `#aabbccff`, and `rgb(170 187 255)` all share the same internal representation, and so are mutually `==` to one another. We test this in the tests for color parsing and constructor functions; here we can safely assume that the internal representations are the same, and that we only need to verify checks for that expected representation.